src/styles/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  background-color: #fff;
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  margin: 40px 16px 0;
+}
+
+.card__code {
+  line-height: 14px;
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin: 4px 16px 0;
+}
+
+.card__reviews {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 16px 16px 0;
+  align-items: center;
+  height: 16px;
+}
+
+.card__reviews-value {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__price {
+  height: 18px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 24px 16px 0;
```

Comment:
The horizontal margin for the price is set to 16px, similar to the title, code, and reviews. To follow the DRY principle and to maintain consistency, consider using a variable or a mixin for the horizontal margins if they are the same across multiple elements.